src/Features/Core/Portable/Common/GlyphExtensions.cs

```diff
@@ -208,7 +208,7 @@ private static Glyph GetGlyph(string tag, ImmutableArray<string> allTags)
             return Glyph.None;
         }
 
-        private static Accessibility GetAccessibility(ImmutableArray<string> tags)
+        public static Accessibility GetAccessibility(ImmutableArray<string> tags)
```

Comment:
```suggestion
        public static Accessibility GetAccessibility(this ImmutableArray<string> tags)
```

very nit here, but I would expect this to be an extension like the others in the file.

Reply:
nooooooooooooooooooooooooooooooooooooo do not make highly specific extension methods on widely used types like this.

Reply:
i'm not a fan of extending random collections.  For example, you could have an ImmutableArray<string> in many different contexts.  Having this .GetAccessibility extension show up for all of those seems wonky to me :)

Reply:
almost all static types called XXXExtensions though _are_ extension methods. Should it move instead?